cli/schemas/config-file.v1.json

```diff
@@ -446,7 +446,6 @@
                 },
                 "command": {
                   "type": "string",
-                  "required": true,
                   "description": "The task to execute"
```

Comment:
Can you update the description to mention that this field is required unless there's `"dependencies"` field specified?

Reply:
Is it required though? Having a task like this is valid in the string form:

```json
{
  "tasks": {
    "foo": ""
  }
}
```

And I guess the same is also fine for the object notation:

```json
{
  "tasks": {
    "foo":  {}
  }
}
```

That's just a task that does nothing.